app/src/forms/auth/middleware/apiAccess.js

```diff
@@ -1,6 +1,6 @@
 const Problem = require('api-problem');
 const basicAuth = require('express-basic-auth');
-const { validate: uuidValidate } = require('uuid');
+const uuid = require('uuid');
```

Comment:
Standardizing on just including the module - "validate" is such a generic name for a method that it's a bit more readable to have the context of what we're validating.